supervisord.conf

```diff
@@ -0,0 +1,147 @@
+[unix_http_server]
+file=/tmp/supervisor.sock   ; (the path to the socket file)
+
+[supervisord]
+;logfile=/tmp/supervisord.log ; (main log file;default.conf $CWD/supervisord.log)
+logfile=/dev/null
+logfile_maxbytes=0        ; (max main logfile bytes b4 rotation;default.conf 50MB)
+logfile_backups=10           ; (num of main logfile rotation backups;default.conf 10)
+loglevel=info                ; (log level;default.conf info; others: debug,warn,trace)
+pidfile=/tmp/supervisord.pid ; (supervisord pidfile;default.conf supervisord.pid)
+nodaemon=false               ; (start in foreground if true;default.conf false)
+minfds=1024                  ; (min. avail startup file descriptors;default.conf 1024)
+minprocs=200                 ; (min. avail process descriptors;default.conf 200)
+user=root		             ;
+
+; the below section must remain in the config file for RPC
+; (supervisorctl/web interface) to work, additional interfaces may be
+; added by defining them in separate rpcinterface: sections
+[rpcinterface:supervisor]
+supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+[supervisorctl]
+serverurl=unix:///tmp/supervisor.sock ; use a unix:// URL  for a unix socket
+
+[program:appwrite]
+command=php /usr/src/code/app/http.php -dopcache.preload=opcache.preload=/usr/src/code/app/preload.php
+autostart=true
+autorestart=true
+priority=5
+stdout_events_enabled=true
+stderr_events_enabled=true
+stopsignal=QUIT
+startretries=10
+stdout_logfile=/var/log/appwrite.log
+stdout_logfile_maxbytes=5000000
+stderr_logfile=/dev/stderr
+stderr_logfile_maxbytes = 0
+
+[program:worker-usage]
+command=worker-usage
+autostart=true
+autorestart=true
+priority=5
+stdout_events_enabled=true
+stderr_events_enabled=true
+stopsignal=QUIT
+startretries=10
+stdout_logfile=/var/log/appwrite.log
```

Comment:
It will be nice and easier to debug specific issues if we'll use a separate log file for each process stdout & stderr logs.

Reply:
I have made the changes.